Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -338,7 +332,9 @@ extension ActivityStreamPanel: UICollectionViewDelegateFlowLayout {
         case .pocket:
             return cellSize
         case .libraryShortcuts:
-            return CGSize(width: min(ASPanelUX.LibraryShortcutsMaxWidth, cellSize.width), height: (cellSize.width/4)+10)
+            let numberofshortcuts: CGFloat = 4
+            let titleSpacing: CGFloat = 10
+            return CGSize(width: min(ASPanelUX.LibraryShortcutsMaxWidth, cellSize.width), height: (cellSize.width/numberofshortcuts) + titleSpacing)
```

Comment:
teeny nit: on `/` I think swift lint will complain about lack of space around math operators